bundles/mapping/mapmodule/plugin/scalebar/publisher/ScalebarTool.js

```diff
@@ -0,0 +1,58 @@
+import { AbstractPublisherTool } from '../../../../../framework/publisher2/tools/AbstractPublisherTool';
+
+const SCALEBAR_TOOL_ID = 'Oskari.mapframework.bundle.mapmodule.plugin.ScaleBarPlugin';
+class ScalebarTool extends AbstractPublisherTool {
+    constructor (...args) {
+        super(...args);
+        this.index = 1;
+        this.group = 'additional';
+        this.config = null;
+    }
+
+    getTool () {
+        return {
+            id: SCALEBAR_TOOL_ID,
+            title: Oskari.getMsg('Publisher2', 'BasicView.maptools.ScaleBarPlugin'),
```

Comment:
Maybe we should bring these localizations over to mapmodule if we want to have the tools here?

Reply:
With this it's kind of all over the place with localizations being under publisher, tools under plugins and the actual import is still done on publisher. Maybe we should have everything under the bundles that have the code (in mapmodule in this case). Best case scenario would be if we had some sort of an API how publisher would be able to hook into these without bundling everything in, even when the publisher is NOT present on the app.

Reply:
Moved these under mapmodule localisations in publishertools segment. At least it's more logical now, maybe not more easily automatically found though...